changed button labels and added placeholder to text fields

hzhu359 opened the pull request:
This is for issue #54 
I changed up the buttons and added a placeholder in the text fields:
![Screen Shot 2020-09-29 at 11 15 48 PM](https://user-images.githubusercontent.com/54654449/94639661-b63bcb80-02aa-11eb-92e5-f4dfa2c9b965.png)
If you have any specific label you want to put on the 'random' button lmk and I'll make those changes

migueldlr commented:
Actually do you think we could make a new section for random coin flip? I think the general pattern is to have each "method" in its own section, and to me adding randomly and adding with a given number of coin flips are two distinct methods

hzhu359 commented:
On it!!!

hzhu359 commented:
Added a new group for the random actions and changed the buttons and text field onPresses and callbacks to correspond to the correct actions
![Screen Shot 2020-10-03 at 5 42 44 PM](https://user-images.githubusercontent.com/54654449/95002258-d9fb5c00-059f-11eb-9fea-a66d32bcd06f.png)
